hooks/templatedataformatter.js

```diff
@@ -24,6 +24,7 @@ module.exports = function (pageMetadata, siteLevelAttributes, pageNameToConfig)
     global_config: getLocalizedGlobalConfig(globalConfig, currentLocaleConfig, locale),
     params: currentLocaleConfig.params || {},
     relativePath,
+    pageMetadata: { pageName },
```

Comment:
I am pretty confident nobody added a custom "pageMetadata" to their templatedataformatter or page config files. We don't even let you override templatedataformatter, so it would have to be adding custom stuff to your page config